src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  const dayPrice = 40;
+  const discount1 = 20;
+  const discount2 = 50;
+  const total = dayPrice * days;
+
+  if (days < 3) {
+    return total;
+  }
 
+  if (days < 7) {
+    return total - discount1;
```

Comment:
Similarly, the condition to check for the minimum number of days to receive the first level of discount should use a constant. For example, `const MIN_DAYS_FOR_LONG_TERM_DISCOUNT = 7;` and then `if (days < MIN_DAYS_FOR_LONG_TERM_DISCOUNT) { ... }`.